packages/sdk-node/src/shared.ts

```diff
@@ -25,6 +25,19 @@ export function proxyUrlToAbsoluteURL(url: string | undefined): string {
   return isProxyUrlRelative(url) ? new URL(url, window.location.origin).toString() : url;
 }
 
+export function getRequestUrl({ request, relativePath }: { request: Request; relativePath?: string }): URL {
```

Comment:
❓ Is this function used somewhere?

Reply:
no. i will drop it. this came back due to revert.